wiki/Developer-Documentation.md

```diff
@@ -109,205 +93,70 @@ application is as follows:
-output:
-
-    $ mkdir -p ~/code/qfs/build/release & cd ~/code/qfs/build/release
-    $ cmake -DCMAKE_BUILD_TYPE:STRING="RelWithDebInfo" ~/code/qfs
-    $ make VERBOSE=1
+To build with verbose output, use the environment variable `VERBOSE=true` to
+build. For example, use `VERBOSE=true make`.
 
 ### `make test` Targets
-The top level `Makefile` has information on how to run an end-to-end test as
-part of `make`. This is useful when you do changes to QFS source and want to
-make sure the core functionality is intact.
-
-    $ cd ~/code/qfs
-    $ make test-release
-
-**Notes**
-1. Since this test invokes the metaserver and chunk servers locally and performs
-   various checks, it may take a couple of minutes to complete.
-1. Also note that if you are running this from a partition that is nearly full,
-   the test may fail. Please refer to `maxSpaceUtilizationThreshold` in
-   [[Configuration Reference]].
-
-### Compiling QFS FUSE
-For building FUSE support for QFS, you will need install the FUSE package on the
-target machines and then compile the source as described above.
-
-1. Install FUSE package
-
-        $ yum install fuse-devel fuse-libs fuse (CentOS)
-        $ apt-get install libfuse-dev           (Debian/Ubuntu)
-        Install OSXFUSE                         (OSX)
-
-1. In `~/code/qfs/CMakeLists.txt`, provide the path to libfuse.so. This is given
-   by the following line in CMakeLists.txt: `SET (Fuse_LIBRARY_DIR <libfuse
-   path>)`
-1. Build the fuse binary
-
-        cd ~/code/qfs/build/release
-        cmake ~/code/qfs
-        make && make install
-
-After the last step qfs_fuse binary will be installed in
-`~/code/qfs/build/release/bin/`
-
-You can use the qfs_fuse binary directly or via /etc/fstab.
-
-1. Direct usage:
-    - Mount using `$ sudo ./qfs_fuse <metaserver>:20000 /mnt/qfs -o allow_other,ro`
-    - Unmount using `$ sudo umount /mnt/qfs`
-1. Editing /etc/fstab to mount automatically at startup:
-    - Create a symlink to qfs\_fuse `$ ln -s <path-to-qfs_fuse> /sbin/mount.qfs`
-    - Add the following line to /etc/fstab:`<metaserver>:20000 /mnt/qfs qfs ro,allow_other 0 0`
-
-Due to licensing issues, you can include FUSE only if it is licensed under LGPL
-or any other license that is compatible with Apache 2.0 license.
+To run qfs tests, use `make test`. This will ensure that all core functionality
+is intact. Note that this test invokes the metaserver and chunk servers locally
+and performs various checks, it may take a couple of minutes to complete. If you
+are running this from a partition that is nearly full, the test may fail. Please
+refer to `maxSpaceUtilizationThreshold` in [[Configuration Reference]].
 
 ### Developing a C++ client
-One can follow the sample C++ client at
-`~/code/qfs/examples/cc/qfssample_main.cc`. The QFS client library API is
-defined in `~/code/qfs/src/cc/libclient/KfsClient.h`.
+To deveop a c++ client, see the sample code in the
```

Comment:
Thank you for catching all my spelling mistakes. I must have written this late at night or something.